app/models/post.rb

```diff
@@ -1,5 +1,26 @@
+# class Post < ApplicationRecord
+#   belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+#   has_many :comments
+#   has_many :likes
+# end
+
 class Post < ApplicationRecord
-  belongs_to :author, class_name: 'User'
-  has_many :likes
+  belongs_to :author, class_name: 'User', foreign_key: 'user_id'
```

Comment:
- [ ] According to the [erd diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png) the user is referenced in the posts table with the name `authorId` so the foreign key reference for this association here should be `author_id` and not `user_id` or you can skip the foreign_key reference here as class_name reference is already there.